### ISSUE TITLE
Airline Issue "Thai AirAsia X"

### ISSUE THREAD

**Issue by tiamsanit** [jpatokal/openflights#560]
Hello,
I think airline database is incorrect.
"Thai AirAsia X" IATA: XJ ICAO: TAX
Country: Thailand, Active now
https://en.wikipedia.org/wiki/Thai_AirAsia_X

In openflight database it has Thai AirAsia X (priv)
and IATA: XJ still belong to inactive Mesaba Airlines.

Best regadrs,

**Comment by rdasilva33** [comment]
Indeed, this is still incorrect almost half a year later. Here's the Wikipedia data about the airline: https://en.wikipedia.org/wiki/Thai_AirAsia_X. It is also not possible to add a new airline because the name already exists! When will this be fixed???

**Comment by rdasilvacuriel** [comment]
@jpatokal, any chance this could be solved any time soon? I'm not sure about your exact data structure, but I can imagine the problem of one airline IATA code per airline can be solved by adding dates to the airlines when they come to life and when they go bust, such that airline IATA codes can be "re-used", belonging to one airline for one period and another airline for another period. Obviously IATA allow code re-use so it is a real-life situation that should be modelled in the database. To keep the change simple there's no need to enter dates for all airlines, but just add them for airlines where it is applicable, like for "Thai Air Asia X". Right now my and other peoples' flights with this airline are listed as "Priv###" instead of XJ### as they should be, and it's not possible to make the change through the website because a previously existing airline already "owns" this code. Many thanks!

**Comment by rdasilva33** [comment]
What a shame! Not even worth a reply? I've removed my flights from this site to https://my.flightradar24.com/ which is far better anyway in my opinion, and they DO have Thai AirAsia X.
